instrumentation/apache-httpclient/apache-httpclient-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v4_0/ApacheHttpClientInstrumentation.java

```diff
@@ -123,171 +126,140 @@ public void transform(TypeTransformer transformer) {
         this.getClass().getName() + "$RequestWithHandlerAdvice");
   }
 
-  @SuppressWarnings("unused")
-  public static class UriRequestAdvice {
+  public static class AdviceScope {
```

Comment:
[for reviewer] this `AdviceScope` class has been moved outside of the Advice classes as it allows to remove duplication, otherwise by default almost identical `AdviceScope` inner classes are created for each advice class.